sunbeam-python/sunbeam/commands/maas.py

```diff
@@ -73,14 +79,37 @@ class RoleTags(enum.Enum):
     CONTROL = "control"
     COMPUTE = "compute"
     STORAGE = "storage"
-    JUJU = "juju"
+    JUJU_CONTROLLER = "juju-controller"
 
     @classmethod
     def values(cls) -> list[str]:
         """Return list of tag values."""
         return [tag.value for tag in cls]
 
 
+ROLE_NETWORK_MAPPING = {
+    RoleTags.CONTROL: [
+        Networks.INTERNAL,
+        Networks.PUBLIC,
+        Networks.STORAGE,
+    ],
+    RoleTags.COMPUTE: [
+        Networks.DATA,
+        Networks.INTERNAL,
+        Networks.STORAGE,
+    ],
+    RoleTags.STORAGE: [
+        Networks.DATA,
+        Networks.INTERNAL,
+        Networks.STORAGE_CLUSTER,
```

Comment:
Storage node need Networks.STORAGE as well right? this is now clients access the Ceph storage from compute nodes or control nodes.

Reply:
Fixed in last commit